lib/ecom/api.ts

```diff
@@ -197,9 +197,31 @@ const createEcomApi = (wixClient: WixApiClient): EcomApi =>
             return wixClient.auth.loggedIn();
         },
         async getUser() {
-            const response = await wixClient.members.getCurrentMember();
+            const response = await wixClient.members.getCurrentMember({
+                fieldsets: [members.Set.FULL],
+            });
             return response.member;
         },
+        async updateUser(id: string, user: members.UpdateMember) {
+            // `updateMember` is not clearing contact phone number,
+            // there is a separate `deleteMemberPhones` function for that
+            // we will handle clear phone number case separately
+            const shouldClearPhoneNumber = user.contact?.phones?.[0] === '';
+            if (shouldClearPhoneNumber) {
+                // sending empty phone number in phones array will result in 400 response
+                user.contact!.phones = [];
+            }
+
+            const updatedUser = await wixClient.members.updateMember(id, user);
+            // if it was requested to clear phone number - use `deleteMemberPhones` function
+            // because `updateMember` does not handle phone deletion
+            if (shouldClearPhoneNumber && updatedUser?.contact?.phones?.[0] !== undefined) {
```

Comment:
this is kind of duplicate condition and comment from the above.. can we do something like this?
```
if (!user.contact?.phones?.[0]) {
  await wixClient.members.deleteMemberPhones(id);
  user.contact!.phones = [];
}

await wixClient.members.updateMember(id, user);
```